.env.example

```diff
@@ -1,10 +1,12 @@
 NEXT_PUBLIC_CHAIN_ID=testing
 NEXT_PUBLIC_CHAIN_NAME=Wasmd Localhost
 NEXT_PUBLIC_CHAIN_BECH32_PREFIX=wasm
-NEXT_PUBLIC_CHAIN_RPC_ENDPOINT=http://localhost:3000/api
+NEXT_PUBLIC_CHAIN_RPC_ENDPOINT=http://localhost:3001/api
 NEXT_PUBLIC_CHAIN_REST_ENDPOINT=http://localhost:1317
+NEXT_PUBLIC_FEE_DENOM="ucosm"
 NEXT_PUBLIC_STAKING_DENOM="ustake"
-NEXT_PUBLIC_SITE_TITLE=CW-DAO
-NEXT_PUBLIC_SITE_ICON_URL="/juno.svg"
+NEXT_PUBLIC_SITE_TITLE=DAO DAO
```

Comment:
Need to quote these strings?

Reply:
Don't think so, seems to work fine. :shrug: